modules/metastorage-cache/src/main/java/org/apache/ignite/internal/metastorage/cache/IdempotentCacheVacuumizer.java

```diff
@@ -113,8 +112,10 @@ public CompletableFuture<Void> startAsync(ComponentContext componentContext) {
                 () -> {
                     if (triggerVacuumization.get()) {
                         try {
-                            vacuumizationAction.accept(hybridTimestamp(clockService.nowLong()
-                                    - (idempotentCacheTtl.value() + clockService.maxClockSkewMillis())));
+                            HybridTimestamp evictionTimestamp = clockService.now()
```

Comment:
Should we use `now()` here, would `current()` be enough?

Reply:
I don't know, it seems like `current()` should be enough